ceno_zkvm/src/e2e.rs

```diff
@@ -289,14 +291,38 @@ pub fn setup_platform(
     stack_size: u32,
     heap_size: u32,
     pub_io_size: u32,
+) -> Platform {
+    setup_platform_inner(preset, program, stack_size, heap_size, pub_io_size, false)
+}
+
+pub fn setup_platform_debug(
```

Comment:
why adding a new function instead of add a new arg

Reply:
Usually we prefer function to be self-explain which avoid "Boolean Parameter Smell" and possess "Self-documenting Code"

For example.

```
setup_platform_inner(..., true)   // what does `true` mean?
setup_platform_inner(..., false)  // is this debug mode or something else
```

Reply:
ok, that make sense, I might do some refactor later